python/hsfs/storage_connector.py

```diff
@@ -369,6 +369,16 @@ def prepare_spark(self, path: Optional[str] = None) -> Optional[str]:
         """
         return engine.get_instance().setup_storage_connector(self, path)
 
+    def connector_options(self) -> Dict[str, Any]:
+        """Return options to be passed to an external S3 connector library"""
+        return {
+            "access_key": self.access_key,
+            "secret_key": self.secret_key,
+            "session_token": self.session_token,
+            "path": self.path,
```

Comment:
what is this path? shouldn't it be received from feature group?

Reply:
for ex:
`fg.location` == 's3://ralfsbucket/fg_delta_1'

Reply:
right, it should be the location of the feature group